### PR TITLE
Update service-admin-portal.md

### PR DESCRIPTION
**Pull request by andreea-s**
I am copying the info box from the guest users switch to the external sharing as well to clarify that guest users are treated as internal after they join.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@andreea-s : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by kfollis** [comment]
#sign-off

**Comment by PRMerger16** [comment]
@andreea-s : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@andreea-s : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/service-admin-portal.md [hunk 1]
```diff
@@ -185,14 +185,17 @@ In the admin portal, you also control which users have permissions to distribute
...
+Users in the organization can share dashboards, reports and apps with users outside the organization. Learn more about [sharing externally](service-share-dashboards.md#share-a-dashboard-or-report-outside-your-organization).
 
 ![External users setting](media/service-admin-portal/powerbi-admin-sharing-external-02.png)
 
 The following image shows the message that appears when you share with an external user.
 
 ![Share with external user](media/service-admin-portal/powerbi-admin-sharing-external.png)  
 
+> [!IMPORTANT]
+> When you add an external user to your Azure AD tenant (either manually or by sharing content with that user), that user account is now discoverable to internal accounts. An internal user can share other things with the external user even if the internal account doesn't have external sharing permissions.
```
Comment (LukaszPawlowski-MS) [inline powerbi-docs/service-admin-portal.md]: I would state it a little differently: This option controls whether users in Power BI can invite external users to become Azure AAD B2B Guest users in your organization through Power BI. When enabled, users who have Guest Inviter role in AAD can add external email addresses when sharing reports, dashboards, or Power BI apps. The external recipient is invited to join your organization as an Azure AAD B2B Guest user.  Importantly, when disabling this setting, external users who are already Azure B2B Guests in your organization continue to appear in people picker UIs in Power BI and can be given access to items, workspaces, or apps.

Comment (andreea-s) [reply powerbi-docs/service-admin-portal.md]: done!

**Comment by PRMerger15** [comment]
@andreea-s : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by andreea-s** [comment]
Once I have this approved, what is the comment for merging? :)

**Comment by PRMerger16** [comment]
You can only sign off in the main PR conversation, not in review comments. In general, the creator of the PR should sign off, not the tech reviewer.

**Comment by PRMerger12** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**